services/api/README.md

```diff
@@ -20,16 +20,33 @@ Install all dependencies from the root of the monorepo:
 npm install
 ```
 
-In the `api` service, you'll want to ensure that you've configured your `.env` file and setup a database. To apply migrations:
+In the `api` service, you'll want to ensure that you've configured your `.env` file and setup a database. Create a Postgres database named `algo-api` then apply migrations:
 
 ```bash
 npm run db:latest
 ```
 
 Per the `.env`, you'll also need to be connected to an Algorand node, whether in development or production.
 
+### Creating a local Algorand Sandbox account
 For local development, the [Algorand Sandbox](https://github.com/algorand/sandbox) is handy docker instance that makes interfacing with the blockchain simple from your local machine. It will also create an account that can be funded with fake Algos using the [Testnet Dispenser](https://dispenser.testnet.aws.algodev.network/).
- 
+
+To create an account:
+* Download the [Alogrand Sandbox] and start up the docker instance:
```

Comment:
Just making it a valid link (if that was the intention):

```suggestion
* Download the [Alogrand Sandbox](https://github.com/algorand/sandbox) and start up the docker instance:
```

Alternatively, remove the brackets:

```suggestion
* Download the Alogrand Sandbox and start up the docker instance:
```